control-plane/subcommand/common/common.go

```diff
@@ -115,3 +118,44 @@ func WriteFileWithPerms(outputFile, payload string, mode os.FileMode) error {
 	}
 	return os.Chmod(outputFile, mode)
 }
+
+// GetHTTPSScheme return the string value "https" if the bool https is true and "http" if it is false.
+func GetHTTPSScheme(https bool) string {
```

Comment:
nit: seems overkill to function this

Reply:
yeah thats fair. It was a fall out of trying to do a larger refactor and then needed to trim it down.